model/config.go

```diff
@@ -589,7 +589,7 @@ func (o *Config) SetDefaults() {
 		*o.LocalizationSettings.DefaultClientLocale = DEFAULT_LOCALE
 	}
 
-	if o.LocalizationSettings.AvailableLocales == nil {
+	if o.LocalizationSettings.AvailableLocales == nil || *o.LocalizationSettings.AvailableLocales == "" {
```

Comment:
not really, if it defaults to blank the we're going to have issues in the frontend.

This way it will always have available all the supported languages.